app/src/main/java/com/piwniczna/mojakancelaria/Models/ClientEntity.kt

```diff
@@ -0,0 +1,14 @@
+package com.piwniczna.mojakancelaria.Models
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "clients")
+data class ClientEntity(
+
+    @ColumnInfo(name = "Name")
+    var name: String ="",
```

Comment:
I remember that we were talking about firstName and lastName, so are we staying with name only?

Reply:
Tak i tak